plugins/auth-backend/src/service/router.ts

```diff
@@ -137,10 +154,5 @@ export async function createRouter({
     }),
   );
 
-  router.use('/:provider/', req => {
-    const { provider } = req.params;
-    throw new NotFoundError(`No auth provider registered for '${provider}'`);
-  });
```

Comment:
This should still be here right? But with the message `Unknown auth provider '${provider}'` (when a junk string is found that we don't know what to do with). It may seem unnecessary but the thing is that this exception will be properly serialized into the complex error response form that the frontend can show in a pretty way, compared to a naked 404 from express

Reply:
Hmm, worth providing a better 404